src/ir/value.h

```diff
@@ -107,6 +107,8 @@ class StoredValue {
   StoredValue(const StoredValue&) = default;
   StoredValue& operator=(const StoredValue&) = default;
 
+  const Storage& storage() const { return *storage_; }
```

Comment:
This change seems a little off-theme for this PR.

Reply:
Yeah, this change is needed for the code to compile. Otherwise, we get an unused field error. All of these get cleaned up in the child PRs though.